### ISSUE TITLE
No icons display for all browsers except Edge

### ISSUE THREAD

**Issue by trulycool** [darktrojan/openwith#115]
I'm on Windows 10, Firefox, 48.02. I have several different browsers installed, and Open With finds them all fine, but only displays an icon for Edge.

This is such a great extension, but the lack of icons, or the ability to at least set them manually is really hampering it's usefulness.

It's OK when using the drop down menu, but ideally I'd have the buttons in the toolbar, available as a single click, and they are useless there as blanks. Well, not exactly useless as they display tooltips, but much less convenient than one would like.

![2016-09-10_12-35-04](https://cloud.githubusercontent.com/assets/821458/18412070/d3533654-7753-11e6-9b0c-201af42e90d3.png)

![2016-09-10_12-35-45](https://cloud.githubusercontent.com/assets/821458/18412072/ec5439f0-7753-11e6-8afb-93b2d515ba70.png)

**Comment by trulycool** [comment]
Started digging into this a bit more, and found the wiki with instructions for manually setting an icon via about:config.

However, manually setting an icon for an entry does not work either. Some part worked as the blank page icon disappeared, so I know I got the key syntax correct. However, I tried *.png, and several different formats of *.ico, as well as the *.exe, and all produced the same result. The blank page placeholder disappeared, the menu text moved over, but no icon shows...

![openwith menu](https://cloud.githubusercontent.com/assets/821458/18421648/2b2f7450-785d-11e6-9ad9-fb36d4da26be.png)

![openwith about-config](https://cloud.githubusercontent.com/assets/821458/18421655/3c008896-785d-11e6-9708-c5b452c92f0d.png)

** Note ** This is the version of the plugin I sent a pull request for, which is why the icon in the toolbar does have a transparent background, but the only change I made was to the toolbar icon, no code changes, and I tested in a stock version too, same results.

**Comment by darktrojan** [comment]
Have you tried disabling other addons and/or your theme to see if they are causing the problem?

**Comment by trulycool** [comment]
Yes, same result. I've just created an empty profile, Firefox 48.0.2, 
with only Open With installed (no other add-ons or themes...

**Note** this is a fresh install of Open With also, not the version 
which I updated the bg transparency of the toolbar icon (and created the 
pull request for).

-M

**Comment by darktrojan** [comment]
I was just looking on Bugzilla for more info and realised I've seen this before, but never found a solution: https://bugzilla.mozilla.org/show_bug.cgi?id=710123
